test/uk/gov/hmrc/channelpreferences/controllers/PreferenceControllerSpec.scala

```diff
@@ -360,9 +364,22 @@ class PreferenceControllerSpec extends PlaySpec with ScalaFutures with MockitoSu
           Future.successful(Left(SERVICE_UNAVAILABLE))
       },
       mockAuthConnector,
+      mockEntityResolverConnector,
       Helpers.stubControllerComponents()
     )
 
   }
+  trait ConfirmGenerator {
+    val entityIgGen: Gen[String] = Gen.uuid.map(_.toString)
+    val itsaIdGen: Gen[String] = Gen.choose(0, 9999999).map(_.toString)
+    val randomWordGen: Gen[String] = Gen.oneOf("foo", "bar", "baz", "fizz", "buzz", "toto", "tata")
+    val httpResponseGen: Gen[HttpResponse] =
+      for {
+        status <- Gen.oneOf(200, 401, 400, 404)
```

Comment:
magic numbers

Reply:
Sorted in https://github.com/hmrc/channel-preferences/pull/28/commits/610773fecbf710d12e2d19adf3e42483cfe78be1